jobs/JGLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -11,12 +18,19 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
+export GDUMP="gdas"
+export GDUMP_ENS="enkf${GDUMP}"
+export RUN="${GDUMP}"
+export OPREFIX="${RUN}.t${cyc}z."
 
 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
```

Comment:
`$RUN` should be considered read-only. It can only be changed internally for `declare_from_tmpl()` like this:
```suggestion
export OPREFIX="${RUN}.t${cyc}z."

RUN="${GDUMP}" YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
```